lib/utils/injectRefreshEntry.js

```diff
@@ -38,13 +37,11 @@ function injectRefreshEntry(originalEntry, options, compiler) {
   let resourceQuery = {};
 
   if (compiler && compiler.options && compiler.options.devServer) {
-    resourceQuery.domain = createDomain(compiler.options.devServer);
```

Comment:
Removed because it wasn’t used at all. Probably has some special meaning inside of `webpack-dev-server/client`, we don’t need that